src/main/java/n2/loops/Task5.java

```diff
@@ -0,0 +1,90 @@
+package n2.loops;
+
+public class Task5 {
+    public static void main(String[] args) {
+        int sum1 = 0;
```

Comment:
you can re-use one variable between different tasks

Reply:
done